Code/Framework/AzToolsFramework/Tests/Prefab/PrefabAssetFixupTests.cpp

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Asset/AssetSerializer.h>
+#include <AzCore/Serialization/Json/JsonUtils.h>
+#include <AzCore/Component/Component.h>
+#include <Prefab/PrefabTestFixture.h>
+
+namespace UnitTest
```

Comment:
Does this test either SetAssetFixUp() or FixUpInvalidAssets?

Reply:
It calls LoadInstanceFromPrefabDom which does that